tools/regexp-generator/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "test262-regexp-class-escapes",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.mjs",
```

Since this is not meant to be published, we could add

```json
"private": true
```

and remove unnecessary fields:

```suggestion
  "private": true,
  "main": "index.mjs",
```

---

i believe name and version are always required, even for private packages

---

Nope, they’re only required when you want to publish the package, which isn’t the case here